slack_bolt/listener/internals.py

```diff
@@ -0,0 +1,76 @@
+from logging import Logger
+from typing import Optional, Dict, Any, List
+
+from slack_bolt.request.request import BoltRequest
+from slack_bolt.response.response import BoltResponse
+
+from slack_bolt.request.payload_utils import (
+    to_options,
+    to_shortcut,
+    to_action,
+    to_view,
+    to_command,
+    to_event,
+    to_message,
+    to_step,
+)
+
+
+def _build_all_available_args(
+    logger: Logger,
+    request: BoltRequest,
+    response: Optional[BoltResponse],
+    error: Optional[Exception] = None,
+) -> Dict[str, Any]:
+    all_available_args = {
+        "logger": logger,
+        "error": error,
+        "client": request.context.client,
+        "req": request,
+        "request": request,
+        "resp": response,
+        "response": response,
+        "context": request.context,
+        "body": request.body,
+        # payload
+        "body": request.body,
```

Comment:
Same suggestion as above. Please disregard if it's on purpose:
```suggestion
        # payload
        "body": request.body,
```